code/style.css

```diff
@@ -49,7 +50,7 @@ p {
 
 button {
   padding: 8px 16px;
-  background: crimson;
+  background: rgb(218, 74, 103);
   border: none;
   font-size: 20px;
   border-radius: 8px;
```

Comment:
This is a general comment for your whole code :) I like your code and didn't find anything to "change" or do different except the comment on getElementById. Good job Liza!